test/e2e/accounts/common.ts

```diff
@@ -196,17 +188,36 @@ async function switchToAccount2(driver: Driver) {
 
 export async function connectAccountToTestDapp(driver: Driver) {
   await switchToOrOpenDapp(driver);
-
   await driver.clickElement('#connectButton');
 
-  await driver.delay(regularDelayMs);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-  await driver.clickElementAndWaitForWindowToClose({
+
+  const edit = await driver.findClickableElements({
+    text: 'Edit',
+    tag: 'button',
+  });
+  await edit[1].click();
+
+  await driver.clickElement({
+    tag: 'p',
+    text: 'Localhost 8545'
+  });
+
+  await driver.clickElement({
+    text: 'Update',
+    tag: 'button',
+  });
+
+  await driver.clickElement({
     text: 'Connect',
     tag: 'button',
   });
 
   await driver.switchToWindowWithUrl(DAPP_URL);
+  await driver.waitForSelector({
+    css: '[id="chainId"]',
+    text: '0x539',
+  });
```

Comment:
we make sure that we continue to be on localhost in the test dapp, after connecting. There's no reason to change networks